stdlib/public/TensorFlow/TensorShape.swift

```diff
@@ -10,6 +10,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+import Python
```

Comment:
Could you please do the following?
* Rename `stdlib/public/TensorFlow/NumpyConversion.swift` to `PythonConversion.swift`.
  * Make appropriate changes to the file header so it's not NumPy-specific.
* Move `import Python` and the `PythonConvertible` logic from this file to `PythonConversion.swift`.

That would be much appreciated! 🙂

Reply:
@dan-zheng sure thing: https://github.com/apple/swift/pull/23891